### DISCUSSION TITLE
Is there any IDE for Move like nujan.io or Remix

### DISCUSSION THREAD

**Discussion by rajgits** [movementlabsxyz/movement#350] [Ideas]
https://ide.nujan.io/ 
see this IDE i am using for Func and Tact deployer so same like  anything available for MOVE?  these IDE will saves the time and will give general view for each contract functions like Getting , Setting request. somehow like Remix.

**Comment by l-monninger** [comment]
There isn't yet. There are a couple of syntax highlighting extensions for VsCode. I personally just work with the compiler when I write Move though.

**Comment by rajgits** [comment]
Okay got it.
